docs/index.html

```diff
@@ -0,0 +1,30 @@
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<title>Financial | Chart.js</title>
+		<script src="moment.js" type="text/javascript"></script>
+		<script src="Chart.bundle.js" type="text/javascript"></script>
```

Comment:
I believe that `Chart.bundle.js` includes moment.js. I think we'd either use just `Chart.bundle.js` or plain `Chart.js` and `moment.js`

Reply:
I tried with Chart.bundle.js but it didn't work
Some people seem to have the save issue : https://stackoverflow.com/q/46766670
I went with `Chart.js` + `moment.js`